modules/script/script-impl/src/test/resources/graaljs/function-script.js

```diff
@@ -0,0 +1,7 @@
+input => {
+     if (input === "undefinedValue") {
+        return undefined;
+     } else {
+        return "Hello, " + input;
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=8692791)
```suggestion
        return 'Hello, ' + input;
```